refine

```diff
@@ -96,23 +96,23 @@ check_running() {
     CHECK_STR="<title>OpenRefine</title>"
 
     if [ "$CURL" ] ; then
-        curl -s -S -f $URL > /dev/null 2>&1
+        curl --noproxy 127.0.0.1 -s -S -f $URL > /dev/null 2>&1
         CURL_RETURN=$?
         if [ $CURL_RETURN -eq "7" ] || [ $CURL_RETURN -eq "22" ]  ; then
             NOT_RUNNING="1"
         fi
     elif [ "$WGET" ] ; then
-        wget -O - $URL > /dev/null 2>&1
+        wget --noproxy 127.0.0.1 -O - $URL > /dev/null 2>&1
```

Comment:
No, the same syntax does not work for wget, as `--noproxy` is not a valid wget option.